bpf/include/bpf/config/node.h

```diff
@@ -15,4 +15,6 @@
 /* Legacy node config rendered at agent runtime. */
 #include <node_config.h>
 
+NODE_CONFIG(__u32, ipv4_loopback, "IPv4 loopback address")
```

Comment:
A few thoughts:
- Paul made a list of [most all macros used in the datapath](https://github.com/cilium/cilium/issues/15379#issuecomment-897441189), I suggest we stick to those wherever possible, or at leat use them as a starting point.
- This can be turned into a `[4]byte` so we can do `node.LoopbackIPv4 = ([4]byte)(lnc.LoopbackIPv4)` below.
- Looking at other variables in the codebase, it's more common to use _ipv4 and _ipv6 as a suffix; don't feel obliged to retain the names of the macros you port over 1:1. For example, there's already `endpoint_ipv4/6` in lxc.h, which flows more naturally when read/pronounced.

Reply:
> Paul made a list of https://github.com/cilium/cilium/issues/15379#issuecomment-897441189, I suggest we stick to those wherever possible, or at leat use them as a starting point.

I'm a bit confused, are you referring to the description, `"IPv4 loopback address"`?

> Looking at other variables in the codebase, it's more common to use _ipv4 and _ipv6 as a suffix; don't feel obliged to retain the names of the macros you port over 1:1. For example, there's already endpoint_ipv4/6 in lxc.h, which flows more naturally when read/pronounced.

Does `loopback_ipv4` sound good to you instead?